src/components/progress/progress.scss

```diff
@@ -0,0 +1,149 @@
+@import "../../themes/ionic.globals";
+
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:               rgb(16, 108, 2001) !default;
```

Comment:
It is currently material only, and there is no separation of `.scss` and `.md.scss`, which I at least feel is needed for it not to be changed after being merged (all the classes names)